lib/game.dart

```diff
@@ -23,6 +23,7 @@ class CoronaJump extends BaseGame with HasWidgetsOverlay {
   LevelWrapper level;
   double maxHeight = 0;
   int score = 0;
+  int localHighScore = 0;
```

Comment:
To keep things clean (and make the code platform agnostic in game and both overlays), can the current highscore be saved in the score helper?

Reply:
@tguelenman, you must have missed this.

Reply:
@abegehr sorry, missed it indeed. Same thing here though, can we just leave it like this? If I'm totally wrong and this project has a future, then I'll fix it with pleasure